src/security/builtin_plugins/authentication/src/authentication.c

```diff
@@ -1473,9 +1400,9 @@ validate_remote_identity(
         DDS_Security_set_token_nil(local_auth_request_token);
     }
 
-    *remote_identity_handle = IDENTITY_HANDLE(remoteIdent);;
+    *remote_identity_handle = IDENTITY_HANDLE(relation);;
```

Comment:
double semi-colon (and since there are some comments anyway, I might as well point it out)